flexget/plugins/input/plex.py

```diff
@@ -149,7 +152,11 @@ def on_task_input(self, task, config):
         entries = []
         if config['unwatched_only'] and config['section'] != 'recentlyViewedShows' and config['section'] != 'all':
             urlconfig['unwatched'] = '1'
-        if config['username'] and config['password']:
+        if config['token']:
+            accesstoken = config['token']
+            log.debug("Using accesstoken: %s" % accesstoken)
```

Comment:
Pass it as argument instead of doing the string formatting explicitly. Code seems fine otherwise.